jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SamProjectGenerator.kt

```diff
@@ -108,29 +97,19 @@ class SamProjectGenerator :
 /**
  * Used to overwrite the entire panel in the "light" IDEs so we don't put our settings under "More Settings"
  */
-class SamProjectRuntimeSelectionStep(
-    projectGenerator: SamProjectGenerator
-) : ProjectSettingsStepBase<SamNewProjectSettings>(
-    projectGenerator,
-    AbstractNewProjectStep.AbstractCallback<SamNewProjectSettings>()
-) {
-    fun getLocationField(): TextFieldWithBrowseButton = myLocationField
-
-    override fun registerValidators() {
-        super.registerValidators()
-        (peer as SamProjectGeneratorSettingsPeer).registerValidators()
-    }
-}
+class SamProjectRuntimeSelectionStep(projectGenerator: SamProjectGenerator) :
+    ProjectSettingsStepBase<SamNewProjectSettings>(projectGenerator, AbstractNewProjectStep.AbstractCallback<SamNewProjectSettings>())
```

Comment:
haven't looked in a while, but i'm thinking we should actually use the callback class to hook onto the project generation phase. might be more effort than it's worth/not actually make sense though

Reply:
Its a static class that calls doProjet after opening the IDE frame so it does hit us eventually.